src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/sidepanel.jelly

```diff
@@ -22,10 +22,11 @@ THE SOFTWARE.
     <l:header />
     <l:side-panel>
         <l:tasks>
+            <j:set var="canManageTemplate" value="${it.hasManagePermission()}"/>
             <l:task href="" icon="symbol-settings"
-                    title="${app.hasPermission(app.MANAGE) ? '%Configure' : '%View Configuration'}"/>
+                    title="${canManageTemplate ? '%Configure' : '%View Configuration'}"/>
             <j:if test="${!it.readonlyFromUi}">
-                <l:delete permission="${app.MANAGE}" title="${%Delete Pod Template}" message="${%delete.template(it.name)}"/>
+                <l:delete permission="${it.CONFIGURE}" title="${%Delete Pod Template}" message="${%delete.template(it.name)}"/>
```

Comment:
I think this is incorrect, this should refer to the manage permission used in the `PodTemplateGroup` implementation.

Reply:
Right. The `l:delete` can be switch to a conditional and a `<l:task/>` within it.